test/e2e/experiment_test.go

```diff
@@ -99,6 +99,23 @@ func (s *ExperimentSuite) TestExperimentWithServiceAndScaleDownDelay() {
 		ExpectExperimentServiceCount("experiment-with-service", 0)
 }
 
+func (s *ExperimentSuite) TestExperimentWithMultiportServiceAndScaleDownDelay() {
+	g := s.Given()
+	g.ApplyManifests("@functional/experiment-with-multiport-service.yaml")
+	g.When().
+		WaitForExperimentPhase("experiment-with-multiport-service", "Running").
+		Sleep(time.Second*5).
```

Comment:
```suggestion
                WaitForExperimentCondition("experiment-with-service", func(ex *rov1.Experiment) bool {
			return s.GetReplicaSetFromExperiment(ex, "test").Status.Replicas == 1
		}, "number-of-rs-pods-meet", fixtures.E2EWaitTimeout).
```

Reply:
@alexef what do you think about this?

Reply:
the proposed change makes sense, most of the time rs gets ready in less than 5 seconds